javascript-loops-practice/odd-or-even.js

```diff
@@ -0,0 +1,11 @@
+/* exported oddOrEven */
+function oddOrEven(numbers) {
+  var newArray = [];
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      numbers[i] = 'even';
+    } else if (numbers[i] !== 0) {
+      numbers[i] = 'odd';
+    } newArray.push(numbers[i]);
```

Comment:
You should not have lines of code on the same lines as closing braces, please move this to its own line, along with all the `return` statements you have on the same line as closing curly braces in the functions you've written in this exercise.